src/idl_namer.h

```diff
@@ -0,0 +1,114 @@
+#ifndef FLATBUFFERS_IDL_NAMER
+#define FLATBUFFERS_IDL_NAMER
+
+#include "flatbuffers/idl.h"
+#include "namer.h"
+
+namespace flatbuffers {
+
+// Provides Namer capabilities to types defined in the flatbuffers IDL.
+class IdlNamer : public Namer {
+ public:
+  explicit IdlNamer(Config config, std::set<std::string> keywords)
+      : Namer(config, std::move(keywords)) {}
+
+  using Namer::Constant;
```

Comment:
why do we need these using statements? Is there trouble with calling the base classes methods somehow?

Reply:
Yeah, because the derived class has the same method name, it hides the base class method. So you have to explicitly list out the base class methods that can be called on the derived class.